pkg/config/config.go

```diff
@@ -80,6 +80,34 @@ type Server struct {
 	CacheDictionarySize int               `deprecated:"true"`
 	CacheSegmentSize    int               `deprecated:"true"`
 	CacheTreeSize       int               `deprecated:"true"`
+
+	// TODO: remove skip: true when we enable these back
+	GoogleEnabled      bool   `skip:"true" def:"false" desc:"enables Google Oauth"`
+	GoogleClientID     string `skip:"true" def:"" desc:"client ID generated for Google API"`
+	GoogleClientSecret string `skip:"true" def:"" desc:"client secret generated for Google API"`
+	GoogleRedirectURL  string `skip:"true" def:"" desc:"url that google will redirect to after logging in. Has to be in form <pathToPyroscopeServer/google/callback>"`
+	GoogleAuthURL      string `skip:"true" def:"https://accounts.google.com/o/oauth2/auth" desc:"auth url for Google API (usually present in credentials.json file)"`
+	GoogleTokenURL     string `skip:"true" def:"https://accounts.google.com/o/oauth2/token" desc:"token url for Google API (usually present in credentials.json file)"`
+
+	GitlabEnabled bool `skip:"true" def:"false" desc:"enables Gitlab Oauth"`
+	// TODO: why is this one ApplicationID and not ClientID ?
```

Comment:
Gitlab calls it Application ID while others call it Client ID, but it's used in the same manner in both cases

<img width="992" alt="Screenshot 2021-07-16 at 22 55 09" src="https://user-images.githubusercontent.com/8813875/126007539-d6a76575-0268-4379-86be-4b910cb88329.png">